src/_locales/pt_BR/messages.json

```
@@ -0,0 +1,104 @@
+{
+    "pluralRule": {
+        "message": "1"
```

Thank you for the update. Are currently `pt` and `pt_BR` identical? 

One main difference I might found is, that Brazilian Portuguese might need `pluralRule = 2` [as mentions here](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals#Plural_rule_2_(2_forms)), while Portuguese [has Plural Rule 1](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals#Plural_rule_1_(2_forms)).

We have [both rules implemented](https://github.com/projectdelphai/panorama-tab-groups/blob/master/CONTRIBUTION.md#add-language).

---

Yes, it should be rule 2. I tried to read some articles about plural rules, to understand better and don't mess the translation. By our definition, Zero has no means of plurality, so it should be singular.

```suggestion
        "message": "2"
```